app/src/test/java/org/kiwix/kiwixmobile/zim_manager/ZimManageViewModelTest.kt

```diff
@@ -337,9 +337,9 @@ class ZimManageViewModelTest {
       )
       expectNetworkDbAndDefault(
         listOf(
-          Book().apply { language = "eng" },
-          Book().apply { language = "eng" },
-          Book().apply { language = "fra" }
+          book(language = "eng"),
+          book(language = "eng"),
```

Comment:
@macgills I don't understand much what is going on, but this duplicate line seems really weird too me.

Reply:
this shows that when there a multiple of the same language in the network result it increments the occurrence of the language. if you look at the full test, specifically the verify
```
 verify {
        newLanguagesDao.insert(
          listOf(
            dbLanguage.copy(occurencesOfLanguage = 2),
            Language(
              active = false,
              occurencesOfLanguage = 1,
              language = "fra",
              languageLocalized = "",
              languageCode = "",
              languageCodeISO2 = ""
            )
          )
        )
      }
```

Reply:
The `.copy` method is a great little addition to a `data class` in kotlin. it allows you to create a new copy of an object but the method takes named parameters for any field of the object so here I only wanted to update `occurencesOfLanguage ` in my new object